### ISSUE TITLE
Players can get "Stagnant Forever" in the Compressed Smoke Field

### ISSUE THREAD

**Issue by LautaroL20** [BEEmod/BEE2-items#1861]
![Image](https://cloud.githubusercontent.com/assets/12244376/25088989/00db3a64-2351-11e7-8f23-04043ff62e0e.jpg)

**Comment by ZombiePwner1** [comment]
@LautaroL20 This also happens with the force deflection field.

**Comment by TheDarkBomber** [comment]
The player should be killed when it closes on them.

**Comment by LautaroL20** [comment]
> The player should be killed when it closes on them.

Indeed, also the trigger kill must be as thin as possible, as well as with a logic duration of less than a half second and after that, the trigger should turn off.

**Comment by Luke18033** [comment]
I don't think it should kill players, that is arguably more realistic but seems like it could lead to a lot of unexpected/unfair deaths for people who aren't aware of that behavior. It should probably use a VScript to detect which side of the field they're closer to, and push them out that way.

**Comment by LautaroL20** [comment]
I must admit that that could also be a reasonable solution to implement.

**Comment by TeamSpen210** [comment]
The main difficulty is ensuring there isn't already stuff in the way - if it's got glass next to it, for example.
